xml/net_sync.xml

```diff
@@ -80,782 +74,369 @@
-    or vi if no editor was defined). The editor call can be circumvented by
-    entering the comment in advance on the command line, such as in the
-    following example:
-   </para>
-<screen>cvs import -m 'this is a test' synchome &exampleuser_plain; &exampleuserII_plain;</screen>
-  </sect2>
+  <!-- Use cases and properties of Rsync? -->
 
-  <sect2 xml:id="sec.net.sync.cvs.using">
-   <title>Using CVS</title>
-   <para>
-    The synchronization repository can now be checked out from all hosts with
-    <command>cvs</command>&nbsp;<option>co synchome</option>. This creates a
-    new subdirectory <filename>synchome</filename> on the client. To commit
-    your changes to the server, change to the directory
-    <filename>synchome</filename> (or one of its subdirectories) and enter
-    <command>cvs</command>&nbsp;<option>commit</option>.
-   </para>
+  <sect2 xml:id="sec.net.sync.rsync.syntax">
+   <title>Basic Syntax</title>
    <para>
-    By default, all files (including subdirectories) are committed to the
-    server. To commit only individual files or directories, specify them as in
-    <command>cvs</command>&nbsp;<option>commit file1 directory1</option>. New
-    files and directories must be added to the repository with a command like
-    <command>cvs</command>&nbsp;<option>add file1 directory1</option> before
-    they are committed to the server. Subsequently, commit the newly added
-    files and directories with <command>cvs</command>&nbsp;<option>commit file1
-    directory1</option>.
+    Rsync is a command line tool which the following basic syntax:
    </para>
+   <screen>rsync [OPTION] SOURCE [SOURCE]... DEST</screen>
+   <para> You can use Rsync to any machine that you have access to and write
+    permissions, be it local or remote. <replaceable>SOURCE</replaceable> can
+    be added one or many times. The <replaceable>SOURCE</replaceable> and
+     <replaceable>DEST</replaceable> placeholders can be paths, URLs, or a
+    combination of these. </para>
    <para>
-    If you change to another workstation, check out the synchronization
-    repository if this has not been done during an earlier session at the same
-    workstation.
-   </para>
-   <para>
-    Start the synchronization with the server with
-    <command>cvs</command>&nbsp;<option>update</option>. Update individual
-    files or directories as in <command>cvs</command>&nbsp;<option>update file1
-    directory1</option>. To see the difference between the current files and
-    the versions stored on the server, use the command
-    <command>cvs</command>&nbsp;<option>diff</option> or
-    <command>cvs</command>&nbsp;<option>diff file1 directory1</option>. Use
-    <command>cvs</command>&nbsp;<option>-nq update</option> to see which files
-    would be affected by an update.
-   </para>
-   <para>
-    Here are some status symbols displayed during an update:
+    The most common options for rsync:
```

Comment:
The most common options for rsync:

**Below are the most common Rsync options**